packages/service/index.js

```diff
@@ -121,6 +121,7 @@ async function loadPlugin (app, config, pluginOptions) {
       if (plugin.__esModule === true) {
         plugin = plugin.default
       }
+      /* c8 ignore next 4 */
```

Comment:
Are 3 of the 4 ignored lines really just closing curly braces 😄

Reply:
Pretty much yes. `c8` is fun sometimes.